docs/prerequisites/feature-flagging/randomization-sdk/server-sdks/java.md

```diff
@@ -0,0 +1,99 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# JAVA
+
+Eppo's JAVA SDK is open source:
+- [GitHub repository](https://github.com/Eppo-exp/java-server-sdk)
+- [Package](https://s01.oss.sonatype.org/#nexus-search;quick~eppo-server-sdk)
+
+### 1. Install the SDK
+In your pom.xml, add the SDK package as a dependency:
+
+```
+<dependency>
+  <groupId>cloud.eppo</groupId>
+  <artifactId>eppo-server-sdk</artifactId>
+  <version>1.0.0</version>
+</dependency>
+```
+
+### 2. Define an Assignment Logger
+
+The SDK requires an assignment logger to be passed on initialization. The SDK invokes the logger to capture assignment data whenever a variation is assigned. The below code example shows how to integrate the SDK with [Segment](https://segment.com/docs/) for logging events. You could also use your own logging system; the only requirement is that the SDK receives a `logAssignment` function.
+
+Define an implementation of the Eppo `IAssignmentLogger` interface. This interface has one function: `logAssignment`.
+
+```java
+from com.eppo.sdk.dto.IAssignmentLogger;
+from com.eppo.sdk.dto.AssignmentLogData;
+
+public AssignmentLoggerImpl implements IAssignmentLogger {
+  public void logAssignment(AssignmentLogData event) {
+    ...
+  }
+}
+```
+
+The SDK will invoke the `logAssignment` function with an `event` object that contains the following fields:
+
+| Field | Description | Example |
+| --------- | ------- | ---------- |
+| `experiment` (string) | An Eppo experiment key | "recommendation_algo" |
+| `subject` (string) | An identifier of the subject or user assigned to the experiment variation | UUID |
+| `variation` (string) | The experiment variation the subject was assigned to | "control" |
+| `timestamp` (Date) | The time when the subject was assigned to the variation | 2021-06-22T17:35:12.000Z |
+| `subjectAttributes` (Map<String, String>) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }` |
+
+### 3. Initialize the SDK
+
+Initialize the SDK once when your application starts up to generate a singleton client instance. During initialization, the SDK does a network request to fetch experiment configurations, which it stores in memory. It's only necessary to initialize the SDK once per application lifecycle.
+
+The below code example shows how to initialize the SDK with the event logger from the previous section and your API key:
+
+```java
+
+package org.example;
+
+import com.eppo.sdk.EppoClient;
+import com.eppo.sdk.dto.EppoClientConfig;
+
+public class App 
```

Comment:
To be concise, just show the SDK code parts without including  `public class App` or `public static void main`

Reply:
Removed the other code now only showing the initialisation code